azure-pipelines.yml

```diff
@@ -133,72 +133,28 @@ extends:
-
-              - task: CMDLine@2
-                displayName: 'Run tests for electron webview'
-                inputs:
-                  script: |
-                    pnpm exec xvfb-maybe playwright test
-                  workingDirectory: '$(AppHostingSdkProjectDirectory)'
-
-              - task: PublishTestResults@2
-                inputs:
-                  testResultsFormat: 'JUnit'
-                  testResultsFiles: '**/e2e-tests-report*.xml'
-                  testRunTitle: 'E2E Tests - Default'
-                  mergeTestResults: true
-                condition: succeededOrFailed()
-
-          - job: E2ETest3
-            displayName: 'E2E Test - Local Tag'
-            timeoutInMinutes: 90
-            pool:
-              name: Azure-Pipelines-1ESPT-ExDShared
-              image: 'ubuntu-latest'
-              os: linux
-            steps:
-              - template: tools/yaml-templates/build-app-host.yml@self
-                parameters:
-                  appHostGitPath: AppHostingSdk
-
-              - task: CmdLine@2
-                displayName: 'Build Test App Local'
-                inputs:
-                  script: |
-                    pnpm build-test-app-local
-                  workingDirectory: '$(ClientSdkProjectDirectory)'
-
-              - bash: 'node tools/cli/runAppsWithE2ETests.js --useDataFromLocal=true --reportFileName=e2e-tests-report-local-script-tag --envType=localScriptTag'
-                displayName: 'Run E2E integration tests with local script tag'
-                condition: succeeded()
-                workingDirectory: '$(AppHostingSdkProjectDirectory)'
-
-              - task: PublishTestResults@2
-                inputs:
-                  testResultsFormat: 'JUnit'
-                  testResultsFiles: '**/e2e-tests-report*.xml'
-                  testRunTitle: 'E2E Tests - Local Tag'
-                  mergeTestResults: true
-                condition: succeededOrFailed()
+          - template: tools/yaml-templates/web-e2e-tests-job.yml@self
+            parameters:
+              AppHostingSdk: AppHostingSdk
+              hostingEnvironmentType: 'standardWeb'
+              teamsJsReferenceType: 'npm'
+              testPrefixPatternGroups: ['{[0-9],[A-D],[a-d]}', '{[E],[e]}', '{[F-M],[f-m]}', '{[N-Z],[n-z]}']
+
+          - template: tools/yaml-templates/web-e2e-tests-job.yml@self
+            parameters:
+              AppHostingSdk: AppHostingSdk
+              hostingEnvironmentType: 'standardWeb'
+              teamsJsReferenceType: 'scriptTag'
+              testPrefixPatternGroups: ['{[0-9],[A-D],[a-d]}', '{[E],[e]}', '{[F-M],[f-m]}', '{[N-Z],[n-z]}']
+
+          - template: tools/yaml-templates/web-e2e-tests-job.yml@self
+            parameters:
+              AppHostingSdk: AppHostingSdk
+              hostingEnvironmentType: 'electron'
+              teamsJsReferenceType: 'npm'
 
           - job: E2ETestCDN
```

Comment:
>E2ETestCDN [](http://example.com/codeflow?start=17&length=10)

There is still opportunity to leverage the new YAML template for running these tests as well. However, since I don't understand how these work and aren't sure how to validate them, I didn't want to mess with them quite yet and I didn't want to delay getting the faster tests out to developers while I spent time figuring this out. This will be a follow-up item for me.